Command not found (PATH does not contain /usr/sbin)

Getting errors when using LISAv3 to deploy and test CentOS 7_9 on Azure

`[ERROR] lisa.env[generated_0].node[0].cmd[7289] not found command: Command not found: modinfo. Check that modinfo is installed and on $PATH`

`[ERROR] lisa.env[generated_0].node[0].cmd[1038] not found command: Command not found: waagent. Check that waagent is installed and on $PATH`

`[ERROR] lisa.env[generated_0].node[0].cmd[8629] not found command: Command not found: lsmod. Check that lsmod is installed and on $PATH`

SSHing into the node confirms that all three of these commands are present and runnable on the node.

The error about modinfo missing appears to occur before any tests start running. These errors do not occur when deploying and testing Ubuntu 18.04-LTS.

@LiliDeng Those commands fallback to use sudo automatically. It needs to set no error level and info level log. If it's a tool, it should be handled at command detection phase, please check why it doesn't work.

```log
2021-09-07 22:37:21.095[3488][DEBUG] lisa.env[generated_0].node[0].cmd[2764] cmd: ['waagent', '-version'], cwd: None, shell: False, sudo: False, posix: True, remote: True
2021-09-07 22:37:21.178[3488][ERROR] lisa.env[generated_0].node[0].cmd[2764] not found command: Command not found: waagent. Check that waagent is installed and on $PATH
2021-09-07 22:37:21.178[3488][DEBUG] lisa.env[generated_0].node[0].cmd[3786] cmd: ['/usr/sbin/waagent', '-version'], cwd: None, shell: False, sudo: False, posix: True, remote: True
2021-09-07 22:37:21.440[15676][DEBUG] lisa.env[generated_0].node[0].cmd[3786].stdout WALinuxAgent-2.2.46 running on centos 7.9.2009
```

--- a/lisa/tools/lsmod.py
+++ b/lisa/tools/lsmod.py
@@ -36,6 +36,7 @@ class Lsmod(Tool):
         no_error_log: bool = True,
     ) -> bool:
         result = self.run(
+            sudo=True,
             force_run=force_run,
             no_info_log=no_info_log,
             no_error_log=no_error_log,

--- a/lisa/tools/modinfo.py
+++ b/lisa/tools/modinfo.py
@@ -30,20 +30,13 @@ class Modinfo(Tool):
     ) -> str:
         result = self.run(
             mod_name,
+            sudo=True,
             force_run=force_run,
             no_info_log=no_info_log,
             no_error_log=no_error_log,
+            expected_exit_code=0,
+            expected_exit_code_failure_message=f"Modinfo failed for module {mod_name}",
         )
-        if result.exit_code != 0:
-            # CentOS may not include the path when started,
-            # specify path and try again.
-            self._command = "/usr/sbin/modinfo"
-            result = self.run(
-                mod_name,
-                force_run=force_run,
-                no_info_log=no_info_log,
-                no_error_log=no_error_log,
-            )
         return result.stdout
 
     def get_version(

